### PR TITLE
enables a way to retrieve the cookie from the http response

### PR DESCRIPTION
**Pull request by kdrtibor**
This is in common because both plugins need this functionality in order to set the cookie for the browser/ webEngine to download/display images.

### CODE REVIEW

File: src/main/java/com/hpe/adm/octane/ideplugins/services/connection/ClientLoginCookie.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
+package com.hpe.adm.octane.ideplugins.services.connection;
```
Comment (vladbenchea) [inline src/main/java/com/hpe/adm/octane/ideplugins/services/connection/ClientLoginCookie.java]: Forgot to add copyrights

File: src/main/java/com/hpe/adm/octane/ideplugins/services/connection/ClientLoginCookie.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        try {
+            HttpRequest httpRequest = requestFactory.buildGetRequest(new GenericUrl(pictureLink));
+            httpRequest.getHeaders().setCookie(lwssoCookie.toString());
+            httpResponse = httpRequest.execute();
+        } catch (IOException e) {
+            logger.error(e.getMessage());
+            return null;
+        }
+        //test http request format when cookie is expired
+        if(httpResponse.getStatusCode() == 403){
```
Comment (vladbenchea) [inline src/main/java/com/hpe/adm/octane/ideplugins/services/connection/ClientLoginCookie.java]: @atothhpe @kdrtibor  question here: Isn't there a case where 408 would happen? 408's description: "The client did not produce a request within the time that the server was prepared to wait. The client MAY repeat the request without modifications at any later time." 
Won't there be a case like this, when having for example a slow connection?

File: src/main/java/com/hpe/adm/octane/ideplugins/services/connection/ExposedGoogleHttpClient.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package com.hpe.adm.octane.ideplugins.services.connection;
```
Comment (vladbenchea) [inline src/main/java/com/hpe/adm/octane/ideplugins/services/connection/ExposedGoogleHttpClient.java]: This class is not needed anymore

File: src/main/java/com/hpe/adm/octane/ideplugins/services/nonentity/ImageService.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
+package com.hpe.adm.octane.ideplugins.services.nonentity;
```
Comment (vladbenchea) [inline src/main/java/com/hpe/adm/octane/ideplugins/services/nonentity/ImageService.java]: Forgot to add copyrights

File: src/main/java/com/hpe/adm/octane/ideplugins/services/connection/ClientLoginCookie.java (hunk 2 above)
Comment (kdrtibor) [reply src/main/java/com/hpe/adm/octane/ideplugins/services/connection/ClientLoginCookie.java]: In our case the server is Octane which does not have a wait time for a request to happen as far as i know.

**Pull request merged**